sympy/parsing/sympy_parser.py

```diff
@@ -1049,6 +1049,14 @@ class EvaluateFalseTransformer(ast.NodeTransformer):
         ast.BitAnd: 'And',
         ast.BitXor: 'Not',
     }
+    functions = [
```

Comment:
If a tuple is used, one can avoid building the list every time, but is can be pre-compiled.